### DISCUSSION TITLE
Resilient looks awesome

### DISCUSSION THREAD

**Discussion by vrilcode** [Omikhleia/resilient.sile#48] [General]
I've been searching for a typesetting system for a book project for months, if not years. I've used LaTex (great ecosystem, but no real grid layout) and ConTeXt (well done and better grid support, but studying the docs is quite painful and costs a lot of time). I've done a little research on web-technology-based typesetting systems, which is a promising approach, but they are far away from typographic quality of print-publishing systems. I've looked at troff (especially groff and neatroff) and related macro packages like mom or ms. I think, this could still be a way to make great, high-quality books, but usage is a little (too?) exotic.

Now I found SILE and especially resilient and it looks really great. So many things done right, great technical decisions. I want to list a few, which I find great: Markdown as more or less native input format (and even Djot, which I didn't know before - looks even more promising; and even this Djot implementation, eg. having substitutions with :...:), styles abstracted in YAML format with inheritance, great looking default settings for books -- and in general: typographic knowledge meets wise technical decisions.

resilient's approach looks incredible and I hope, that I can realise my book project with this software. I'am really looking forward to additional functionality. Unfortunately I am not a SILE/Lua developer, otherwise I would help to implement side notes or citeproc/CSL integration.

Thanks for this great piece of software. Hope it spreads and others will find it too.

**Comment by Omikhleia** [comment]
Thanks for the feedback and support, it's appreciated!
There are a number of features and improvements I have in mind for the resilient classes, to address my own typesetting needs better as I use them. While they cannot cover everything, by nature, I hope other will find these classes and packages interesting too!
